src/fosslight_util/write_excel.py

```diff
@@ -82,18 +82,29 @@ def remove_empty_sheet(sheet_items):
     return success, final_sheet_to_print
 
 
-def get_header_row(sheet_name, sheet_content):
+def get_header_row(sheet_name, sheet_content, extended_header):
     selected_header = []
+
```

Comment:
I suggest the following. 
- AS-IS :
```
    if extended_header:
        for header_key in extended_header.keys():
            _HEADER[header_key] = extended_header[header_key]
```
- TO-BE: https://pypi.org/project/jsonmerge
```
from  jsonmerge import merge
result = merge(_HEADER, extended_header)
```

Reply:
For TO-BE, `from jsonmerge import merge` should be added at the top of this file.
And `jsonmerge` should be added to requirements.txt.

Reply:
Changed as suggested.